packages/web-app-files/src/store/getters.ts

```diff
@@ -32,9 +32,11 @@ export default {
   totalFilesCount: (state, getters) => {
     const fileCount = getters.filesAll.filter((file) => file.type === 'file').length
     const folderCount = getters.filesAll.filter((file) => file.type === 'folder').length
+    const spaceCount = getters.filesAll.filter((file) => file.type === 'space').length
```

Comment:
Is this sufficient? As far as I can tell we don't want to count personal spaces for example. Which would mean that you need to extend the filter callback to also check the driveType.

Reply:
Good catch, t hx!